oshdb-api/src/test/java/org/heigit/ohsome/oshdb/api/tests/TestMapReduce.java

```diff
@@ -20,7 +20,7 @@
 import org.junit.Test;
 
 /**
- * Test basic map-reduce method of the OSHDB API.
+ * Base class for testing the map-reducer backend implementations of the OSHDB API.
  */
 abstract class TestMapReduce {
```

Comment:
Is “base class” the correct term here? I would have assumed that a base class doesn't contain tests which are actually executed. But in this case the class operates as base class **and** as a normal test class, correct?

Reply:
Hmm, how would you call it? Since the class itself is abstract it can't test anything by itself. Maybe just something like this is better:

```java
  * Test cases for testing the map-reducer backend implementations of the OSHDB API.
```

Reply:
Nevermind, I misinterpreted the output of my IDE.